### PR TITLE
Gradpath Prototype

### PR DESCRIPTION
**Pull request by mistergone**
This pull request replaces the existing Grad Path tool with the latest iteration.

The changes are contained wholly within the GradPath code, and entails changes to templates (content), as well as changes to the JavaScript to accommodate changes to the flow of the application. We have also added two new React components that fit into the existing code - one for the **progress stepper**, and one for the **tooltips**.

---
## Additions
- Code for program stepper and tooltips

## Changes
- Modify the way navigation buttons work
- Add code to support "net price" content
- Add code to support "costs not covered loan" content
- Moved most content around
- Changed code to allow for multiple instances of graphs/charts
- Add new analytics events

## Checklist
- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Future todos are captured in comments and/or tickets
- [x] Project documentation has been updated, potentially one or more of:

#### Browser testing
Visually tested in the following supported browsers:
- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] ~Internet Explorer 11 and 8 (via emulation in 11's dev tools)~ (We are not supporting ie11/8)
- [x] Safari on iOS
- [x] Chrome on Android

#### Accessibility

- [x] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [x] Screen reader friendly
- [x] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [ ] ~~Is useable without CSS~~ (this app is not useable without CSS)
- [ ] ~~Is useable without JS~~ (this app is not useable without JS)
- [ ] Does not introduce new lint warnings
- [x] Flexible from small to large screens

### CODE REVIEW

**Comment by anselmbradford** [comment]
Should I point https://github.com/cfpb/consumerfinance.gov/pull/8376 to this branch instead of `main`?

**Comment by wpears** [comment]
>Should I point #8376 to this branch instead of main?

Possibly! Things are a bit fluid right now, so maybe wait to update that til this PR lands in main

File: cfgov/unprocessed/apps/paying-for-college/js/college-costs.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+const root = createRoot(document.getElementById('react-stepper'));
+root.render(<Wrapper />);
+
+const tooltipRoot = createRoot(document.getElementById('react-tooltip'));
+tooltipRoot.render(
+  <Tooltip
+    id="a-tooltip"
+    border="1px solid var(--gray-40)"
+    opacity="1"
+    clickable={true}
```
Comment (contolini) [inline cfgov/unprocessed/apps/paying-for-college/js/college-costs.jsx:102]:
```suggestion
    clickable={true}
    globalCloseEvents={{ escape: true }}
```

to be compliant with https://designsystem.digital.gov/components/tooltip/accessibility-tests/ (we just went through this with TCCP)

**Review by wpears — APPROVED** [review]
This needs content updates, but approving the foundation so @mistergone can merge it in when we're ready

**Pull request merged**